Automattic/woocommerce-payments
# Update phpcs and related sniffs

## Changes proposed in this Pull Request









* Update the WooCommerce and Slevomat PHPCS sniffs so we can update PHPCS itself.
* Update PHPCS so it can be run via PHP 8.1.

#### Testing instructions





> [!NOTE]
> There are currently _way more issues_ reported after the update. Not sure which ones are false positives, which can be silenced, and which ones we should fix.

1. Run `npm run lint:php` in `develop` and make a note of the issues reported (there should be none).
    - If you're already running PHP v8.1 you may have to run this using something like `/opt/homebrew/opt/php@7.4/bin/php ./vendor/bin/phpcs --standard=phpcs.xml.dist $(git ls-files | grep .php$)`
3. Checkout this branch and make sure you run `composer install`.
4. Run `npm run lint:php` and make sure you see the same issues reported as in `develop`.

-------------------


**Post merge**
